components/desktop/gst-plugins-base1/patches/01.filio.patch

```diff
@@ -0,0 +1,12 @@
+--- gst-plugins-base-1.8.1/gst/tcp/gstmultifdsink.c.orig	2016-09-05 18:52:34.896956799 +0200
++++ gst-plugins-base-1.8.1/gst/tcp/gstmultifdsink.c	2016-09-05 18:54:45.854211639 +0200
+@@ -118,6 +118,9 @@
+ #include <sys/stat.h>
+ #include <netinet/in.h>
+ 
++#if defined (__sun) && defined(__SVR4)
++#define HAVE_FIONREAD_IN_SYS_FILIO
```

Comment:
I do not know...
Considering that our Gobject introspection is broken with multilib would you mind merging with either disabling introspection for 32bit of both?